src/me/StevenLawson/TotalFreedomMod/Commands/Command_tfm.java

```diff
@@ -65,6 +65,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 TotalFreedomMod.buildCreator), ChatColor.GOLD);
         playerMsg("Running on " + TFM_ConfigEntry.SERVER_NAME.getString() + ".", ChatColor.GOLD);
         playerMsg("Created by Madgeek1450 and Prozza.", ChatColor.GOLD);
+        playerMsg("If you wish to contribute to the development of this plugin, go to " + ChatColor.AQUA + "http://github.com/TotalFreedom/TotalFreedomMod.", ChatColor.GREEN);
```

Comment:
How about? This plugin is opensource: https://github.com/TotalFreedom/TotalFreedomMod